apps/api/jwt_auth/handlers.py

```diff
@@ -74,14 +83,24 @@ def jwt_decode_handler(token, get_api_key=APIKey.get_jwt_key):
             leeway=api_settings.JWT_LEEWAY,
             algorithms=[api_settings.JWT_ALGORITHM]
         )
+    except jwt.MissingRequiredClaimError, exc:
+        raise exceptions.AuthenticationFailed(
+            detail=u'Invalid JWT: {}'.format(exc))
```

Comment:
Would it be better to log the exception and leave the `AuthenticationFailed` message as something generic like `"Invalid JWT"`?